metrics/heapster.go

```diff
@@ -121,14 +121,13 @@ func createAndRunAPIServer(opt *options.HeapsterRunOptions, metricSink *metricsi
 		glog.Errorf("Could not create the API server: %v", err)
 		return
 	}
-	healthz.InstallHandler(server.Mux, healthzChecker(metricSink))
```

Comment:
> Also, your main looks to be fighting with some of the API server endpoints like heath

That shouldn't be the case, since we're on a different port